test/controllers/detailedvaluationrequest/DvrControllerSpec.scala

```diff
@@ -26,22 +25,39 @@ import org.mockito.Mockito.{never, verify, when}
 import org.scalacheck.Arbitrary.arbitrary
 import play.api.test.FakeRequest
 import play.api.test.Helpers._
+import uk.gov.hmrc.govukfrontend.views.Layouts
+import uk.gov.hmrc.govukfrontend.views.html.components._
 import utils.{StubPropertyLinkConnector, _}
+import views.html.dvr._
 
+import java.time.LocalDateTime
 import scala.concurrent.Future
 
-class DvrControllerSpec extends VoaPropertyLinkingSpec {
+class DvrControllerSpec extends VoaPropertyLinkingSpec with Layouts {
 
   trait Setup {
     implicit val request = FakeRequest()
 
+    val realMainLayout = new views.html.mainLayout(
+      GovukTemplate,
+      GovukHeader,
+      GovukFooter,
+      GovukBackLink,
+      GovukDetails,
+      new views.html.head())
```

Comment:
Is it worth pulling this out into a trait so it can be reused?

Reply:
that's a good question. was thinking the same. I am happy to try this, but is it worth the effort?